packages/shared/lib/clients/big-query.client.ts

```diff
@@ -0,0 +1,72 @@
+import { BigQuery } from '@google-cloud/bigquery';
+import logger from '../logger/console.js';
+import { isCloud, isLocal, getEnv } from '../utils/utils.js';
+
+type Row = Record<string, string | number | undefined>;
+
+class BigQueryClient {
+    private client: BigQuery;
+    private datasetName: string;
+    private tableName: string;
+
+    constructor() {
+        this.client = new BigQuery();
+        this.datasetName = 'dataset';
```

Comment:
let's call it `raw`. Right now we would be querying `raw` tables directly but soon we are going to want to aggregate, clean, process those dataset tagles into something that can be displayed in dashboard, etc...